src/explore-education-statistics-common/src/components/form/index.ts

```diff
@@ -1,4 +1,4 @@
-export { default as Form } from './Form';
+// export { default as Form } from './Form';
```

Comment:
Apologies, same comment again really, any reason this is commented out?

Reply:
Just my terrible messiness, it's restored now.